Makefile

```diff
@@ -181,10 +181,6 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
-.PHONY: lint
-lint:
-	@echo 'make lint is depricated. Use "make revive" if you want to use the old lint tool, or "make golangci-lint" to run a complete code check.'
-
```

Comment:
I'd like to eventually bring this `lint` target back that would run all linters (go,js,css).

Reply:
> 
> 
> I'd like to eventually bring this `lint` target back that would run all linters (go,js,css).

Sorry, now I've read this. I'd say go for it! 😄